src/TLuaInterpreter.cpp

```diff
@@ -5869,6 +5869,39 @@ int TLuaInterpreter::tempButton(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::setButtonStyleSheet(lua_State* L)
+{
+    //args: name, css text
+    QString name, css;
+
+    if (!lua_isstring(L, 1)) {
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #1 type (name as string expected, got %s!)", luaL_typename(L, 1));
+        return lua_error(L);
+    } else {
+        name = QString::fromUtf8(lua_tostring(L, 1));
+    }
+    if (!lua_isstring(L, 2)) {
+        lua_pushfstring(L, "setButtonStyleSheet: bad argument #2 type (css as string expected, got %s!)", luaL_typename(L, 2));
+        return lua_error(L);
+    } else {
+        css = QString::fromUtf8(lua_tostring(L, 2));
+    }
+
+    Host& host = getHostFromLua(L);
+    auto actionsList = host.getActionUnit()->findActionsByName(name);
+    if (actionsList.empty()) {
+        lua_pushnil(L);
+        lua_pushfstring(L, "setButtonStyleSheet: no button named \"%s\" found", name.toUtf8().constData());
+        return 2;
+    }
+    for (auto action : actionsList) {
+        action->css = css;
+    }
+    host.getActionUnit()->updateToolbar();
+    lua_pushboolean(L, 1);
```

Comment:
Since we may potentially modify more than one `TAction` __directly__ here is there any merit in returning `actionsList.count()` as a number rather than just a lua `true` on finding at least one match?

Reply:
We don't do that for any other parts of the API - adding that would bring in inconsistency. It may seem like a "good idea" but why introduce known inconsistencies, when there's no demand for it either - we've got the `exists()` function already if you want to count how many items you've got.

Reply:
OK, "good idea" to be dropped in a hole...

         🕳

Reply:
...err hand on a moment `exists()` you say, does that actually ___count___ things then - I wouldn't have guessed that! 😕

Reply:
Yeah we had that discussion when introducing `tempKey()`. Anyway, this is @Mudlet/lua-interface  stuff...